scripts/purge_cache.rb

```diff
@@ -45,8 +45,8 @@ def purge_identifiers_from_cache(options, identifiers)
 
 parser = OptionParser.new do |opts|
   opts.banner = "Usage: #{File.basename($0)} [options]"
-  opts.on("-h", "--hostname", String, "Cantaloupe service hostname")
-  opts.on("-p", "--port", String, "Cantaloupe service port")
+  opts.on("-h", "--hostname [HOSTNAME]", String, "Cantaloupe service hostname")
+  opts.on("-p", "--port [PORT]", String, "Cantaloupe service port")
```

Comment:
This is from https://github.com/NYPL/cantaloupe/pull/49/files (for https://jira.nypl.org/browse/DR-2706), but it looks like that ticket was marked as Done on 1/31:
<img width="984" alt="Screenshot 2024-02-22 at 12 35 02 PM" src="https://github.com/NYPL/cantaloupe/assets/4614468/efb5f4a9-21a0-4693-913f-a8fc01a45a0a">

Is it weird that these changes are included in here? (sorry if this is a dumb question)

Reply:
Yeah I was wondering the same.

Reply:
Yes this is fine to merge, and not a dumb question at all, I'm happy for the attention to detail to make sure we know what we're merging!

After merging DR-2706 and marking the ticket as done, it was found to not quite work for several reasons. 1) we don't have permissions to run the script on the actual prod machines, so I ran it locally instead, but then found 2) there is a bug in the script that required the changes you see here.

So I made the changes, and ran the script locally, and the task was done. But getting that fix out to prod is helpful. The script is not part of the app runtime so it will have no effect, and we still are not able to run it on the prod machines anyway. 

Probably to really do this correctly and less confusingly, I should have made another bug ticket and applied the fix there, so then when the time came to deploy it (now!) we wouldn't be confused.